src/components/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { ContentPlaceholder } from '../ContentPlaceholder';
+import { Context, GAUGE_RADIUS } from '../../constants';
+import styles from './Gauge.scss';
+
+const { block, elem } = bem('Gauge', styles);
+
+interface Props {
+    context?: Context;
+    children?: ReactNode;
+    isProgressLoading?: boolean;
+    isContentLoading?: boolean;
+    percentage: number;
+    value: NotEmptyReactNode;
```

Comment:
If value is a ReactNode, it means that you are leaving the styling for the application? Wouldn't that possibly lead to inconsistencies?